tests.js

```diff
@@ -138,6 +150,21 @@ module.exports = testCase({
       test.done()
     },
 
+    'new Bandit with array should create the arms initialized properly': function(test) {
+
+      var bandit = new Bandit([{count: 100, sum: 20},{count: 50, sum: 30},{count:70, sum: 10}])
```

Comment:
Another minor stylistic thing but is there a way to keep the lines under 80 chars?